packages/reports/tests/dummy/config/environment.js

```diff
@@ -64,11 +64,24 @@ module.exports = function (environment) {
           type: 'elide',
         },
         {
-          name: 'elideTwo',
-          displayName: 'Elide Two',
-          description: 'Elide Two Description',
+          name: 'elideTwoDefault',
+          displayName: 'Elide Two Default',
+          hide: true,
           uri: 'https://data2.naviapp.com/graphql',
           type: 'elide',
+          namespaces: [
+            {
+              name: 'elideTwo',
+              displayName: 'Elide Two',
+              description: 'Elide Two Description',
+            },
+            {
+              name: 'elideTwoCopy',
+              displayName: 'Elide Two Copy',
+              description: 'Copy of Elide Two Description',
+              hide: true,
```

Comment:
What is the case for telling the UI about the namespace but then hiding it? Could removing then namespace entry be just as effective?

Reply:
Theoretically yes, you could just remove the namespace, but IFF the data source you're trying to hide is actually a namespace. This feature is more specifically targeted at the case of trying to hide a default data source while keeping the namespaces under it (like we're trying to do with Digits+).
As a bonus, though, this should also make it easy to suspend data sources in the future - if you wanted to stop people from making new reports/dashboards, but didn't necessarily want to migrate the items already relying on it, you could just hide the source and basically pause production on it without having to worry about backwards compatibility for already existing things